server/src/domain/user/user.service.ts

```diff
@@ -120,6 +121,14 @@ export class UserService {
     return this.userCore.getUserProfileImage(user);
   }
 
+  async getUserProfileImageHash(userId: string): Promise<string> {
```

Comment:
Can you add tests for this service method?

Reply:
I reverted the changes regarding profileImageHash, as its not needed. I think a test is not needed then.